dapp/src/queries/useTransactionHistoryQuery.js

```diff
@@ -0,0 +1,15 @@
+import { useQuery } from 'react-query'
+
+import { QUERY_KEYS } from '../constants/queryKeys'
+
+import { transactionHistoryService } from '../services/transaction-history.service'
+
+const useTransactionHistoryQuery = (account, options) => {
+  return useQuery(
+    QUERY_KEYS.TransactionHistory(account),
```

Comment:
Since react-query keys are global we need to make sure we don't overwrite them. So I'm maintaining a constant list in `queryKeys`.

It's also very handy to have these keys become you can invalidate queries based on their key.